src/prac.js

```diff
@@ -0,0 +1,23 @@
+front() {
```

Comment:
I'm not sure what to review here - this looks like it is an incomplete solution?